plugins/inputs/sql/sql.go

```diff
@@ -83,7 +83,11 @@ func (q *Query) parse(acc telegraf.Accumulator, rows *dbsql.Rows, t time.Time) (
 			if q.MeasurementColumn != "" && name == q.MeasurementColumn {
 				var ok bool
 				if measurement, ok = columnData[i].(string); !ok {
-					return 0, fmt.Errorf("measurement column type \"%T\" unsupported", columnData[i])
+					var fieldValue []uint8
+					if fieldValue, ok = columnData[i].([]uint8); !ok {
+						return 0, fmt.Errorf("measurement column type \"%T\" unsupported", columnData[i])
+					}
+					measurement = string(fieldValue)
 				}
```

Comment:
Can you please use a type-switch here as it will be much cleaner I think
```go
				switch raw := columnData[i].(type) {
				case string:
					measurement = raw
				case []byte:
					measurement = string(raw)
				case []uint8:
					measurement = string(raw)
				default:
					return 0, fmt.Errorf("measurement column type \"%T\" unsupported", columnData[i])
				}
```